templates/components/status.html.twig

```diff
@@ -1,16 +1,45 @@
-{% macro status(text, options) %}
-    {% set _color = options.color ?? 'green' %}
-    {% set _animated = options.animated ?? true %}
-    {% set _lite = options.lite is defined ? options.lite  : false %}
-    {% set _with_dot = options.with_dot is defined ? options.with_dot  : false %}
-    {% set _extraClass = options.extraClass ?? '' %}
-
-    {%- apply spaceless -%}
-        <span class="status status-{{ _color }} {{ _lite ? 'status-lite' : '' }} {{ _extraClass }}">
-        {% if _with_dot %}
-            <span class="status-dot {{ _animated ? 'status-dot-animated' : '' }}"></span>
-        {% endif %}
-            {{ text }}
-        </span>
-    {%- endapply -%}
-{% endmacro %}
+{% macro status(text, options) %}
+    {% import '@Tabler/components/status.html.twig' as macro %}

```

Comment:
We can maybe rename `macro` to `self`, because macro is already a name in twig.
```diff
-    {% import '@Tabler/components/status.html.twig' as macro %}
+    {% import '@Tabler/components/status.html.twig' as self %}

```

Reply:
Not needed: you can always reference macros in the same file via `_self.status_dot()`

Reply:
I thought about it.
But the fact that devs will call the macro with:
```twig
{% from'@Tabler/components/status.html.twig' import status %}
```
I don't know if `_self` in that macro will be:
1. The devs current twig context 
2. OR the Bundle component context.

To be safe, I've done it hits way.